app/src/lib/stores/helpers/branch-pruner.ts

```diff
@@ -183,7 +183,7 @@ export class BranchPruner {
     }
 
     // Get all branches checked out within the past 2 weeks
-    const twoWeeksAgo = moment().subtract(2, 'weeks').toDate()
+    const twoWeeksAgo = new Date(Date.now() - toMilliseconds(14, 'days'))
```

Comment:
Seems this pattern is really common. Wondering if a helper function would make the code a little more readable?
Like dateAgo(value, unit) => new Date(Date.now() - toMilliseconds(value, unit)) 😄  Tho maybe there is a better name..

Reply:
Yeah, I was thinking about it but I had trouble coming up with a succinct name. I'm super open to it but lets deal with that in a separate PR as an improvement on top of this.

I think your suggestion of `fromNow` is great but only for future dates obviously and `beforeNow` doesn't have quite the same ring to it. Maybe `offsetFromNow(-14, 'days')` or something and we could have an `offsetFrom(date, 24, 'housr')` for when we don't want to use now()?

Reply:
Given all examples we have here use `Date.now()`, I'd rather have something like `offsetFromNow(-14, 'days')`

Reply:
Cool, I'll open a new PR to simplify things with an offsetFromNow function, thanks y'all!